contrib/commit-name.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 last_tag=$(git describe --tags --abbrev=0)
```

Comment:
`git ls-files '*.sh'` is a better bet, as otherwise you'll pick up scripts from dependencies